course/grades.py

```diff
@@ -295,43 +308,195 @@ def grade_key(entry):
         })
 
 
+class ExportGradeBookForm(StyledForm):
+    def __init__(self, *args, **kwargs):
+        # type: (*Any, **Any) -> None  # noqa
+        super(ExportGradeBookForm, self).__init__(*args, **kwargs)
+
+        self.fields["user_info_fields"] = forms.ChoiceField(
+            choices=(csv_handler.export_csv_fields_options),
+            initial=csv_handler.export_csv_fields_options[0],
+            label=_("User Fields to Export"),
+        )
+
+        self.fields["exclude_instructors"] = forms.BooleanField(
+            required=False,
+            initial=True,
+            label=_("Exclude Grades of Instructors"),
+        )
+
+        self.fields["exclude_tas"] = forms.BooleanField(
+            required=False,
+            initial=True,
+            label=_("Exclude Grades of Teaching assistants"),
```

Comment:
"Teaching assistant" and "Instructor" are not concepts in Relate. That means you may not use them in any UI. (They used to be, but they no longer are.) Right now, there are just arbitrarily-defined roles. The only thing available is a "permission" that says whether a role's/participant's grades are to be included in the grades.

Reply:
Ok.